x-pack/plugins/observability_solution/observability_ai_assistant/public/service/create_chat_service.ts

```diff
@@ -137,19 +138,26 @@ export async function createChatService({
   signal: setupAbortSignal,
   registrations,
   apiClient,
+  scope,
 }: {
   analytics: AnalyticsServiceStart;
   signal: AbortSignal;
   registrations: ChatRegistrationRenderFunction[];
   apiClient: ObservabilityAIAssistantAPIClient;
+  scope: AssistantScope;
 }): Promise<ObservabilityAIAssistantChatService> {
   const functionRegistry: FunctionRegistry = new Map();
 
   const renderFunctionRegistry: Map<string, RenderFunction<unknown, FunctionResponse>> = new Map();
 
   const [{ functionDefinitions, systemMessage }] = await Promise.all([
-    apiClient('GET /internal/observability_ai_assistant/functions', {
+    apiClient('GET /internal/observability_ai_assistant/{scope}/functions', {
```

Comment:
should we change the path here to something less specific to the obs ai assistant?

Reply:
Any suggestions? `/internal/ai_assistant/scope/functions`?

Reply:
yeah, might be good to discuss with Serena?